### PR TITLE
serialization util

### PR DESCRIPTION
**Pull request by adlrocha**
This PR adds a simple test to output the cbor serialization in hex of a type implementing the `CborMarshaler`. This comes really handy when debugging serialization issues in the Go-Rust interop with actors.

### CODE REVIEW

**Comment by adlrocha** [comment]
Current dependencies on/for this PR:
* main
  * **PR #8** <a href="https://app.graphite.dev/github/pr/consensus-shipyard/go-ipc-types/8" target="_blank">Graphite</a> 
    * **PR #9** <a href="https://app.graphite.dev/github/pr/consensus-shipyard/go-ipc-types/9" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/consensus-shipyard/go-ipc-types/9?utm_source=stack-comment).

File: utils/serialization_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+
+	"github.com/consensus-shipyard/go-ipc-types/ipcsdk"
+	"github.com/consensus-shipyard/go-ipc-types/subnetactor"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/test-go/testify/require"
```
Comment (dnkolegov) [inline utils/serialization_test.go:11]: not sorted

**Review by dnkolegov — APPROVED** [review]
LGTM!

**Review by ranchalp — APPROVED** [review]
:+1:

**Comment by adlrocha** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: v9fPNHjs9vDydE3qLYu5.
See details on [graphite.dev](https://app.graphite.dev/github/pr/consensus-shipyard/go-ipc-types/9)

**Pull request merged**